packages/fhirman/helpers/sessionLogger.ts

```diff
@@ -3,15 +3,16 @@
 // it is implemented using the singleton pattern
 
 import { createWriteStream, WriteStream } from "fs";
-import { mkdir } from 'node:fs/promises';
+import { mkdir } from "node:fs/promises";
 
-const basePath = "/workspace/sessions/";
+const projectRoot = Deno.env.get("PROJECT_CWD");
```

Comment:
is `PROJECT_CWD` a _free_ env var or do we need to add this to the ENV when running?

Reply:
It was already set!